tests/functional/test_check.py

```diff
@@ -119,7 +119,7 @@ def test_check_complicated_name_missing(script: PipTestEnvironment) -> None:
 
     # Without dependency
     result = script.pip("install", "--no-index", package_a_path, "--no-deps")
-    assert "Successfully installed package-A-1.0" in result.stdout, str(result)
+    assert "Successfully installed package_A-1.0" in result.stdout, str(result)
```

Comment:
maybe we should do a regex match so the pip test suite is not dependent on setuptools>=69 ?

Reply:
or a simple `or`

Reply:
Haha, I totally forgot about that. I'm glad you picked this up :slightly_smiling_face: